### ISSUE TITLE
Invalid material redirect tconstruct:tinkers_bronze as material tconstruct:bronze does not exist

### ISSUE THREAD

**Issue by Insane96** [SlimeKnights/TinkersConstruct#4842]
### Minecraft Version

1.18.1

### Forge Version

36.1.2

### Mantle Version

1.8.37

### Tinkers' Construct Version

3.4.2.60

### Describe your issue

Whenever I open the game / a server the logs mention

`Invalid material redirect tconstruct:tinkers_bronze as material tconstruct:bronze does not exist`

### Crash Report

Actually log: https://pastebin.com/raw/Jhy9RWRE

### Other mods

    BetterAdvancements-1.18.1-0.1.2.122.jar
    BorderlessWindow-1.18-1.4.0.jar
    catalogue-1.6.0-1.18.1.jar
    configured-1.5.3-1.18.1.jar
    CraftTweaker-forge-1.18.1-9.0.81.jar
    create-mc1.18.1_v0.4f.jar
    flywheel-forge-1.18-0.6.1.jar
    Global Data-  Resourcepacks-forge-1.18.1-1.11.3.jar
    IguanaTweaksReborn-2.9.3-mc1.18.1.jar
    ilikewood-1.18.1-7.4.4.0.jar
    InsaneLib-1.4.3-mc1.18.1.jar
    inventorysorter-1.18-19.0.0.jar
    jei-1.18.1-9.4.1.168.jar
    JEITweaker-1.18.1-3.0.0.5.jar
    JustEnoughResources-1.18.1-0.14.0.157.jar
    MouseTweaks-forge-mc1.18-2.21.jar
    shutupexperimentalsettings-1.0.4-1.18+.jar

### Tried reproducing with just Tinkers?

~I will go do that now.~ Yes

### Performance Enchancers

None of the above

### Searched for known issues?

Searched open issues, Searched closed issues, Checked the FAQ

**Comment by KnightMiner** [comment]
> I will go do that now

Did you end up testing?

----

In any case, this is not entirely a bug, mostly there is not a good way to migrate. For a short time during 1.18 before amethyst bronze existed, I merged bronze and tinkers bronze, so tinkers bronze migrated to bronze. When amethyst bronze was added, bronze was made conditional again, but its still the best target for tinkers bronze.

What I might do is migrate tinkers bronze conditionally to either bronze, or if bronze does not exist to some other material to remove the error. JSON syntax already supports that, so it will ensure if you play with bronze, you get the more comparable material, but if you play without your tool is at least not broken.
